CONTRIBUTING.rst

```diff
@@ -129,8 +129,12 @@ To contribute, you will first have to run through some steps:
 - Set up your environment by forking the repository ((`Github doc on
   forking and
   cloning <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo>`__)).
-- While at the root of your local copy of skrub, install the required
-  development dependencies by running
+- Create a new virtual environment either with
```

Comment:
should I add more detail?

Reply:
as they're quite short I would say yes, this one someone following along can just copy-paste without needing to follow the links